Sources/XCBuildSupport/XcodeBuildSystem.swift

```diff
@@ -128,15 +133,19 @@ public final class XcodeBuildSystem: SPMBuildCore.BuildSystem {
         }
 
         guard result.exitStatus == .terminated(code: 0) else {
-            throw Diagnostics.fatalError
-        }
-        
-        if !hasStdout {
-            if !stderrBuffer.isEmpty {
-                diagnostics.emit(StringError(String(decoding: stderrBuffer, as: UTF8.self)))
+            if hasStdout {
```

Comment:
So if there is output on both stdout and stderr, it looks as if stdout "wins"in this case, and stderr doesn't get emitted at all.  Is that on purpose?  Is it desirable?

Reply:
Yep, that's what was already implemented in #3584, I'm just moving the code around. My understanding of rdar://75751711 which that PR fixed was that we only want to emit `stderr` if we got not other output.

Reply:
Interesting — thanks for clarifying.  We might want to revisit this when we have better ways of showing the output in the console, but it is of course fine to leave things as they are for this PR.  Thanks.